l10n_ch_base_bank/models/bank.py

```diff
@@ -298,8 +332,6 @@ def onchange_acc_number_set_swiss_bank(self):
                 bank or
                 self.env['res.bank'].search([('bic', '=', 'POFICHBEXXX')],
                                             limit=1))
-            if not bank.is_swiss_post():
-                self.acc_number = self._get_acc_name()
```

Comment:
This is needed. It is a bit counter intuitive, because at this point, `acc_type` is `postal` do to the entry of the user in `acc_number` which updated `acc_type`. And after we have changed the `acc_number` it might become `bank`.

For instance how it was working in previous versions:

1. Fill `acc_number` with 10-8060-7
2. Onchange on `acc_number` changes `acc_type` to `postal`
3. Onchange on `acc_number` + `acc_type` starts
  3a. on detection of a 'postal' account type search for a bank that has the CCP.
  3b. It matches with BAS bank
  3c. change `acc_number` to a generated name.
  3d. set `bank_id`
  3e. copy the `ccp` that was entered in `acc_number` in `ccp` field.
4. As `acc_number` changed this should trigger onchange on `acc_number` again to set `acc_type` = `bank`